api/modules/food/constants.py

```diff
@@ -7,5 +7,6 @@
 ORIGINAL_ZOMATO_API = ""
 ZOMATO_API_KEY = os.environ.get("ZOMATO_API", ORIGINAL_ZOMATO_API)
 BASE_URL = "https://developers.zomato.com/api/v2.1/geocode?lat={0}&lon={1}"
+BASE_URL_ID = "https://developers.zomato.com/api/v2.1/restaurant?res_id={0}"
```

Comment:
Change `BASE_URL` to `https://developers.zomato.com/api/v2.1/` and then have a `list` API and a `restaurant` API

Reply:
okay